internal/services/cdn/cdn_frontdoor_endpoint_resource.go

```diff
@@ -180,7 +181,7 @@ func resourceCdnFrontDoorEndpointDelete(d *pluginsdk.ResourceData, meta interfac
 	ctx, cancel := timeouts.ForDelete(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorEndpointID(d.Id())
+	id, err := parse.FrontDoorEndpointIDInsensitively(d.Id())
```

Comment:
this is coming from the config and so should be case-sensitive

```suggestion
	id, err := parse.FrontDoorEndpointID(d.Id())
```